lib/ecto/type.ex

```diff
@@ -677,6 +697,9 @@ defmodule Ecto.Type do
       iex> cast(:string, [1, 2, 3])
       :error
 
+      iex> cast(:uuid, "240d1a54-2f3d-43a4-85b9-3ef5ac702feb")
+      {:ok, "240d1a54-2f3d-43a4-85b9-3ef5ac702feb"}
+
```

Comment:
Every other change in this file appears to stem from `mix format` config changes.

Lines 700-702 above are the only intended changes for this commit.